packages/notify-client/src/controllers/engine.ts

```diff
@@ -773,27 +814,44 @@ export class NotifyEngine extends INotifyEngine {
     const notifyKeys = await this.resolveKeys(this.client.notifyServerUrl);
 
     // Derive req topic from did.json
-    const notifyServerWatchTopic = await this.getNotifyServerWatchTopic(
+    const notifyServerWatchReqTopic = await this.getNotifyServerWatchTopic(
       notifyKeys.dappPublicKey
     );
 
     this.client.logger.info(
       "watchSubscriptions >",
       "notifyServerWatchTopic >",
```

Comment:
Just aligning the logged label with what's in source:

```suggestion
      "notifyServerWatchReqTopic >",
```